src/tasks/minions/miningActivity.ts

```diff
@@ -81,24 +79,34 @@ export default class extends Task {
 
 		if (numberOfMinutes > 10 && ore.nuggets) {
 			const numberOfNuggets = rand(0, Math.floor(numberOfMinutes / 4));
-			loot[12012] = numberOfNuggets;
+			loot.add('Golden nugget', numberOfNuggets);
 		} else if (numberOfMinutes > 10 && ore.minerals) {
 			let numberOfMinerals = 0;
 			for (let i = 0; i < quantity; i++) {
 				if (roll(ore.minerals)) numberOfMinerals++;
 			}
 
 			if (numberOfMinerals > 0) {
-				loot[21341] = numberOfMinerals;
+				loot.add('Unidentified minerals', numberOfMinerals);
 			}
 		}
 
-		str += `\n\nYou received: ${await createReadableItemListFromBank(this.client, loot)}.`;
+		if (ore.gems) {
+			for (let i = 0; i < quantity; i++) {
+				loot.add(ore.gems!.roll());
+			}
+		} else {
+			loot.add(ore.id, quantity);
+		}
+		str += `\n\nYou received: ${await createReadableItemListFromBank(
```

Comment:
i think you only need this for gem rock since every other mineable item just gives you the thing youve mined, no need to tell the user twice what they got in that case

Reply:
this message is after the if and else statement and is used for both mineral types

Reply:
ohh ok i didnt see that the diff was saying you moved it, made me think it was added